lib/cloudant.js

```diff
@@ -948,7 +942,9 @@ Cloudant.prototype.updateIndex = function(mo, modelName, cb) {
     },
   };
   /* eslint-enable camelcase */
-  debug('Cloudant.prototype.updateIndex modelName %j', modelName);
+  var indexView = util.inspect(idx, 4);
+  debug('Cloudant.prototype.updateIndex modelName %s idx %s', modelName,
```

Comment:
Still didn't change the string formatting to be more readable:
`'Cloudant.prototype.updateIndex -- modelName: %s, idx: %s'`